docs/trussels.md

```diff
@@ -9,7 +9,7 @@ This is the `Trussels' Guide to Truss`, written by Trussels for Trussels. We sha
 
 > **Disclaimer**: Hi, friend! Thanks for caring so much about this that you are reading the disclaimer! We super-duper appreciate it. Our lawyers need you to know that the Trussels' Guide to Truss is a reflection of Trussels' personal experiences, and neither Truss nor any of its employees are making any promises about anything. In other words, we (that’s all of us) hereby disclaim any and all warranties, duties or guarantees and this document is presented “AS IS”. You assume any and all risks if you incorporate or adopt any of these practices. Also, we make no promises about merchantability, fitness for a particular purpose, lack of viruses, accuracy or completeness of responses, results, workmanlike effort, lack of negligence, or whether reading this a loud at midnight after the correct eldritch sacrifices may summon an unknowable horror that devours souls and flesh alike. We have no idea and make no promises! Whatever happens is all on you. Hooray!
 
-> **Disclaimer for Trussels**: This is a living document and should be considered malleable and fallible. The Founders have not read this document in a long time, and certainly do not read every edit. It really is not an employee handbook or formal statement of policy. If you have any questions about this guide or want to know Truss’s policy on any of these issues, please contact Jen or another Founder. If something seems like it’s wrong, or missing, or whatever, please suggest a change or ask about it in #onboarding on Slack. You can see when it was last updated in the version history.
+> **Disclaimer for Trussels**: This is a living document and should be considered malleable and fallible. The Founders have not read this document in a long time, and certainly do not read every edit. It really is not an employee handbook or formal statement of policy. If you have any questions about this guide or want to know Truss’s policy on any of these issues, please contact Jen or another Founder. If something seems like it’s wrong, or missing, or whatever, please suggest a change or ask about it in #onboarding on Slack. You can also edit the guide directly (["Everyone's GitHub Guide to Updating the Trussels' Guide"] https://docs.google.com/document/d/1gHQehHRlgHh5MrOUmM6DMcTz6EFNQE1HcgNiE-whh8g/edit#heading=h.qzrl50vb7ejp). You can see when it was last updated in the version history.
```

Comment:
This is great. Two small changes requested: 1: can you clean up the the doc URL by removing everything after `/edit`, unless you mean for the link to go to a specific document heading (in which case just say so and it'll go through). The other request is I think this doc isn't public; we have a convention of putting a `:lock:` emoji in front of private links so external readers don't get confused about why the link doesn't work for them.

Reply:
Oops- I made the doc public- the link should now work for any trussel.